services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -461,6 +464,15 @@ public LinkedAccountsResource linkedAccounts() {
         return new LinkedAccountsResource(session, request, client, auth, event, user);
     }
 
+    @Path("/groups")
+    @GET
+    @NoCache
+    @Produces(MediaType.APPLICATION_JSON)
+    public Stream<GroupRepresentation> groupMemberships(@QueryParam("briefRepresentation") @DefaultValue("true") boolean briefRepresentation) {
+        auth.requireOneOf(AccountRoles.MANAGE_ACCOUNT, AccountRoles.VIEW_GROUPS);
```

Comment:
I think this should be view-groups only, as an admin may want to allow folks to manage their accounts, but not be able to view the groups they belong to.

Reply:
I have changed it as you request